Headlines/src/ViewAnimator/Direction.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Direction.swift
+//  Pods-ViewAnimator_Example
+//
+//  Created by Marcos Griselli on 10/5/17.
+//
+
+import UIKit
+
+/// Direction of the animation used in AnimationType.from.
+public enum Direction: Int, CaseIterable {
+
+    case top
+    case left
+    case right
+    case bottom
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewAnimator/Direction.swift#L17"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>Direction.swift:17</code></td>
    </tr>
  </tbody>
</table>